README.md

```diff
@@ -123,6 +128,9 @@ vagrant_plugin 'vagrant-berkshelf'
 end
 
 # Install the plugins as the `donuts` user, into ~/donuts/.vagrant.d
+# .vagrant.d will be allocated if it does not exist.
+# If a specific user, group or mode is desired use a directory resource to
+# create the .vagrant.d directory.
 vagrant_plugin 'vagrant-aws'
```

Comment:
Should this be `vagrant_plugin 'vagrant-aws' do` ?

Reply:
Duh. thanks